app/assets/v2/js/base.js

```diff
@@ -173,7 +173,7 @@ $(document).ready(function() {
 });
 
 $(window).scroll(function() {
-  var scrollPos = $(document).scrollTop();
+  var scrollPos = jQuery(document).scrollTop();
```

Comment:
Can we stick to `$`?  
That's how it's being used everywhere else

cc @SaptakS

Reply:
$ is not `jQuery` on YGE pages... i'm working to fix this in my other active PR (the YGE refactor).. but until that ships, `$` will cause an error if we leave it as is